tcms/rpc/api/testexecution.py

```diff
@@ -229,7 +235,7 @@ def add_link(values, update_tracker=False, **kwargs):
             isinstance(tracker, KiwiTCMS):
         tracker.add_testexecution_to_issue([link.execution], link.url)
 
-    return response
+    return link.serialize()
```

Comment:
Use model_to_dict() b/c the legacy serializer is deprecated and will be removed. That's why it isn't used in the first place.

Reply:
Updated in the second commit.